src/app/components/form-viewer/form-viewer.component.scss

```diff
@@ -83,6 +83,7 @@ $selected-nave-menu-item-color: #424242;
   margin-left: 4px;
   margin-bottom: -8px; //removes the space for input form hint
   width: 100%;
+  font-family: sans-serif;
```

Comment:
Let fonts be handled at the global level in styles.scss so it's clear where they are being set from and all related font settings are together and consistent. If you need to have different fonts/fotn familys (e.g., monospace vs standard) then extend from there. E.g.

.monospace-font 
.sans-serif-font

then @extend .sans-serif-font.

Reply:
This is for the textbox, it defaults to monostpace, and I assume that's not what the user wants.